<launch>
  <node name="task1" pkg="floribot_robot" type="task1_node" >
    <rosparam file="$(find floribot_robot)/config/task1.yaml" command="load" />
  </node>
</launch>